src/script/media/MediaDevicesHandler.ts

```diff
@@ -111,16 +118,27 @@ export class MediaDevicesHandler {
       if (this.availableDevices[deviceType]().length === 0) {
         return '';
       }
-      const isAvailable = this.availableDevices[deviceType]().find(
-        device =>
-          ((device as MediaDeviceInfo).deviceId || (device as ElectronDesktopCapturerSource).id) === currentDeviceId,
-      );
-      if (isAvailable) {
+      const isAvailable = this.availableDevices[deviceType]().find(device => {
+        return (
+          ((device as MediaDeviceInfo).deviceId || (device as ElectronDesktopCapturerSource).id) === currentDeviceId
+        );
+      });
+
+      const isFavorite = loadValue<String[]>(`${deviceType}-fave`)?.find(id => id === currentDeviceId);
```

Comment:
naming feels weird here. `is*` indicates that it's a boolean, but it seems to be a device object. 

Maybe we could rename those `availableDevice` and `favoriteDevice` , WDYT?